src/hapi/pipelines/utilities/mappings.py

```diff
@@ -32,6 +32,8 @@ def get_code_from_name(
     code = code_mapping.get(name_clean)
     if code:
         return code, name_clean, False
+    if len(name) <= 5:
```

Comment:
Maybe put this as a constant at the top of the file?

Reply:
Good idea!